# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux-firmware/no-rdfind.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

We do not really appreciate a quite randomly non standard dependency
just to avoid a handful of duplicate files, ...

--- linux-firmware-20231111/copy-firmware.sh.vanilla	2023-11-16 15:34:09.287635368 +0100
+++ linux-firmware-20231111/copy-firmware.sh	2023-11-16 15:34:53.001638543 +0100
@@ -69,10 +69,12 @@
 	exit 1
 fi
 
+if false; then
 if ! which rdfind 2>/dev/null >/dev/null; then
 	echo "ERROR: rdfind is not installed"
 	exit 1
 fi
+fi
 
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^(RawFile|File):' WHENCE | sed -E -e 's/^(RawFile|File): */\1 /;s/"//g' | while read k f; do
@@ -87,6 +89,7 @@
     fi
 done
 
+if false; then
 $verbose "Finding duplicate files"
 rdfind -makesymlinks true -makeresultsfile false "$destdir" >/dev/null
 find "$destdir" -type l | while read -r l; do
@@ -94,6 +97,7 @@
 	$verbose "Correcting path for $l"
 	ln -fs "$(realpath --relative-to="$(dirname "$(realpath -s "$l")")" "$target")" "$l"
 done
+fi
 
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^Link:' WHENCE | sed -e 's/^Link: *//g;s/-> //g' | while read f d; do
